src/shared_gui_components/MeasureManager.cpp

```diff
@@ -981,9 +981,6 @@ void MeasureManager::checkForRemoteBCLUpdates() {
     int result = msg.exec();
     if (result == QMessageBox::Yes) {
       remoteBCL.updateMeasures();
-      for (auto& oldMeasure : oldMeasures) {
-        LocalBCL::instance().removeMeasure(oldMeasure);
-      }
```

Comment:
So that's the actual fix? There's similar code a dozen lines below though.

Reply:
If `remoteBCL.updateMeasures();` works correctly then it will remove the outdated measures.  However, that wasn't working reliably due to the issue you fixed in https://github.com/NREL/OpenStudio/pull/5129.  My guess is that I was seeing a case where the old measures weren't removed so I added this code in.  However, this code fails when the new measures aren't fetched for some reason, the old ones will just be cleared.  I assumed that is what was happening in the bug report.